node/pkg/logscribeconsumer/app.go

```diff
@@ -0,0 +1,220 @@
+package logscribeconsumer
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"time"
+
+	errorsentinel "bisonai.com/orakl/node/pkg/error"
+	"bisonai.com/orakl/node/pkg/utils/request"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func New(options ...AppOption) (*App, error) {
+	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
+
+	c := &AppConfig{
+		StoreInterval:     DefaultLogStoreInterval,
+		Buffer:            DefaultBufferSize,
+		LogscribeEndpoint: DefaultLogscribeEndpoint,
+		Level:             DefaultMinimalLogStoreLevel.String(),
+	}
+	for _, option := range options {
+		option(c)
+	}
+
+	resp, err := request.RequestRaw(request.WithEndpoint(c.LogscribeEndpoint))
+	if err != nil || resp.StatusCode != http.StatusOK {
+		return nil, errorsentinel.ErrLogscribeConsumerEndpointUnresponsive
+	}
```

Comment:
one thing we should think of is that if we're going to get the logs also from kf node, if not, we need to have another option to use logscribe or not. if we're willing to get from external node outside the k8s cluster, need to get some support from infra team so that we can make the endpoint accessible through tailscale

Reply:
hmm in that case we should have another field in the db to specify which node (kf or our) sent the log, right? I'll discuss this with the infra team and make a separate PR (i think this one is already getting quite big 😄 )

Reply:
I think we can add something like `WithOnlyConsoleLog` or `WithoutDBStorage`

Reply:
no worries, it's not too big, please include the update so that it can be deployed after merged

Reply:
`WithPostToLogscribe` has been implemented which will be taken from the`.env` `POST_TO_LOGSCRIBE` variable. If it's not provided the default is `true`